roles/opencast-config/tasks/main.yml

```diff
@@ -10,10 +10,11 @@
     force: yes
   with_items:
     - { src: "custom.properties", dest: "{{ oc_config_dir }}/custom.properties" }
-    - { src: "org.opencastproject.organization-mh_default_org.cfg", dest: "{{ oc_config_dir }}/org.opencastproject.organization-mh_default_org.cfg" }
+    - { src: "org.opencastproject.organization-mh_default_org.cfg", dest: "{{ oc_config_dir }}/org.opencastproject.organization-mh_default_org.cfg", skip: "{{ 'allinone' in group_names }}" }
```

Comment:
This is problematic for those deploying an all-in-one but not building an AMI.  What about something like like

`skip: "{{ 'allinone' in group_names and build_ami | default(false) }}"`?

Reply:
I thought the default org.opencastproject.organization-mh_default_org.cfg that is installed by the RPM would work fine for an allinone node, it's not reliant on whether it's an AMI or not.

Reply:
While it might work, at the very least it could lead to confused users since the `port` setting in that file won't match the actual configuration.  It defaults to 8080, and these playbooks create a reverse proxy in front of OC, so it should be -1...